src/WingetCreateTests/WingetCreateTests/UnitTests/UpdateCommandTests.cs

```diff
@@ -347,52 +347,51 @@ public async Task BlockUpdateSubmissionsWithNoUpdate()
+        /// Verifies that an error message is shown if multiple architectures are specified for an override.
         /// </summary>
         /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
         [Test]
-        public async Task UpdateFailsOverrideWithMultipleArchitectures()
+        public async Task UpdateFailsOverrideWithMultipleScopes()
         {
             string installerUrl = $"https://fakedomain.com/{TestConstants.TestExeInstaller}";
             (UpdateCommand badCommand, var manifests) =
-                GetUpdateCommandAndManifestData("TestPublisher.ArchitectureOverride", "1.2.3.4", this.tempPath, new[] { $"{installerUrl}|x86|ARM" });
+                GetUpdateCommandAndManifestData("TestPublisher.ScopeOverride", "1.2.3.4", this.tempPath, new[] { $"{installerUrl}|user|machine" });
             var failedUpdateManifests = await RunUpdateCommand(badCommand, manifests);
-            ClassicAssert.IsNull(failedUpdateManifests, "Command should have failed due to multiple architecture overrides specified for a single installer.");
+            ClassicAssert.IsNull(failedUpdateManifests, "Command should have failed due to multiple scope overrides specified for a single installer.");
             string result = this.sw.ToString();
-            Assert.That(result, Does.Contain(Resources.MultipleArchitectureOverride_Error), "Failed to show multiple architecture overrides error.");
+            Assert.That(result, Does.Contain(Resources.MultipleScopeOverride_Error), "Failed to show multiple scope overrides error.");
         }
 
         /// <summary>
-        /// Verfies that an error message is shown if multiple architectures are specified for an override.
+        /// Verifies that an error message is shown if multiple architectures are specified for an override.
         /// </summary>
         /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
         [Test]
-        public async Task UpdateFailsOverrideWithMultipleScopes()
+        public async Task UpdateFailsWithMultipleDisplayVersions()
         {
             string installerUrl = $"https://fakedomain.com/{TestConstants.TestExeInstaller}";
             (UpdateCommand badCommand, var manifests) =
-                GetUpdateCommandAndManifestData("TestPublisher.ScopeOverride", "1.2.3.4", this.tempPath, new[] { $"{installerUrl}|user|machine" });
+                GetUpdateCommandAndManifestData("TestPublisher.ScopeOverride", "1.2.3.4", this.tempPath, new[] { $"{installerUrl}|3.4|1.2" });
             var failedUpdateManifests = await RunUpdateCommand(badCommand, manifests);
-            ClassicAssert.IsNull(failedUpdateManifests, "Command should have failed due to multiple scope overrides specified for a single installer.");
+            ClassicAssert.IsNull(failedUpdateManifests, "Command should have failed due to multiple display versions specified for a single installer.");

```

Comment:
If you take my suggestion above, this error output will be different so be sure to update this test as well.